_dev/src/views/reporting-page.vue

```diff
@@ -40,10 +39,18 @@ export default {
       await this.$store.dispatch('productFeed/GET_PRODUCT_FEED_SETTINGS');
       await this.$store.dispatch('smartShoppingCampaigns/GET_SSC_LIST');
     },
+    async connectAccounts() {
+      await this.$store.dispatch('accounts/REQUEST_GOOGLE_ACCOUNT_DETAILS');
+      await this.$store.dispatch('productFeed/GET_PRODUCT_FEED_SETTINGS');
+    },
   },
   created() {
     if (!this.inNeedOfConfiguration) {
       this.getDatas();
+    } else {
+      this.connectAccounts().then(() => {
+        this.getDatas();
+      });
```

Comment:
```suggestion
  async created() {
    if (this.inNeedOfConfiguration) {
      await this.connectAccounts();
    }
    this.getDatas();
```

Would this work as well? Looks like we have a shorter version